### ISSUE TITLE
Handle higher memory needs for CellAssign

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/scpca-nf#1347]
We had apparently gotten overconfident for memory usage for CellAssign and had it only using the `mem_128` label. We need to revert that to `mem_max`, at least for now. 

I also have a branch where I would like to test using a smaller-than-default batch size in the `model.train()` step to see if that improves memory performance (and maybe overall performance, or at least doesn't slow things much). It would be nice to be able to use less memory for these processes.
